mathics/core/expression.py

```diff
@@ -243,7 +243,8 @@ def do_format(self, evaluation, form):
             include_form = False
             if head in formats and len(self.get_leaves()) == 1:
                 expr = self.leaves[0]
-                if not (form == 'System`OutputForm' and head == 'System`StandardForm'):
+                # removing 'True' below breaks the 'RGBColor[1,0,0]//StandardForm' test cases.
+                if True or not (form == 'System`OutputForm' and head == 'System`StandardForm'):
```

Comment:
Not sure if this was meant to make it into the PR, I think if you're removing the condition completely it'd be better to just remove it.